--- a/.circleci/ecr_gc_docker/docker_hub.py
+++ b/.circleci/ecr_gc_docker/docker_hub.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from collections import namedtuple
 
--- a/.circleci/ecr_gc_docker/gc.py
+++ b/.circleci/ecr_gc_docker/gc.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 import argparse
 import datetime
--- a/.circleci/ensure-consistency.py
+++ b/.circleci/ensure-consistency.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import os
 import subprocess
--- a/.circleci/generate_config_yml.py
+++ b/.circleci/generate_config_yml.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 """
 This script is the source of truth for config.yml.
--- a/.circleci/scripts/build_android_gradle.sh
+++ b/.circleci/scripts/build_android_gradle.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 set -eux -o pipefail
 
 export ANDROID_NDK_HOME=/opt/ndk
--- a/.circleci/scripts/publish_android_snapshot.sh
+++ b/.circleci/scripts/publish_android_snapshot.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 # DO NOT ADD 'set -x' not to reveal CircleCI secret context environment variables
 set -eu -o pipefail
 
--- a/.circleci/scripts/setup_ci_environment.sh
+++ b/.circleci/scripts/setup_ci_environment.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 set -ex -o pipefail
 
 # Set up NVIDIA docker repo
--- a/.circleci/scripts/setup_linux_system_environment.sh
+++ b/.circleci/scripts/setup_linux_system_environment.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 set -eux -o pipefail
 
 # Set up CircleCI GPG keys for apt, if needed
--- a/.circleci/validate-docker-version.py
+++ b/.circleci/validate-docker-version.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 import cimodel.data.caffe2_build_definitions as caffe2_build_definitions
 import cimodel.data.pytorch_build_definitions as pytorch_build_definitions
 from yaml import load
--- a/.jenkins/pytorch/build-mobile-code-analysis.sh
+++ b/.jenkins/pytorch/build-mobile-code-analysis.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 # DO NOT ADD 'set -x' not to reveal CircleCI secret context environment variables
 set -eu -o pipefail
 
--- a/.jenkins/pytorch/build-mobile.sh
+++ b/.jenkins/pytorch/build-mobile.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 # DO NOT ADD 'set -x' not to reveal CircleCI secret context environment variables
 set -eu -o pipefail
 
--- a/.jenkins/pytorch/win-test-helpers/download_image.py
+++ b/.jenkins/pytorch/win-test-helpers/download_image.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 import os
 import sys
--- a/.jenkins/pytorch/win-test-helpers/run_python_nn_smoketests.py
+++ b/.jenkins/pytorch/win-test-helpers/run_python_nn_smoketests.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import print_function
 
--- a/.jenkins/pytorch/win-test-helpers/upload_image.py
+++ b/.jenkins/pytorch/win-test-helpers/upload_image.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 import os
 import sys
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/configure.py
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/configure.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/deps/clog/configure.py
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/deps/clog/configure.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-android-arm64.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-android-arm64.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-android-armv7.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-android-armv7.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-android-x86.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-android-x86.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-arm64.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-arm64.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-arm64e.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-arm64e.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-armv7.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-armv7.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-armv7s.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-armv7s.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-i386.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-i386.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-x86_64.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-ios-x86_64.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-local.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/build-local.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/test-android-arm64.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/test-android-arm64.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/test-android-armv7.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/test-android-armv7.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/test-android-x86.sh
+++ b/aten/src/ATen/native/quantized/cpu/qnnpack/scripts/test-android-x86.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 #
 # Copyright (c) Facebook, Inc. and its affiliates.
 # All rights reserved.
--- a/binaries/bench_gen/bench_gen.py
+++ b/binaries/bench_gen/bench_gen.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import absolute_import
 from __future__ import division
--- a/caffe2/contrib/gloo/gloo_test.py
+++ b/caffe2/contrib/gloo/gloo_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import absolute_import
 from __future__ import division
--- a/caffe2/core/nomnigraph/op_gen.py
+++ b/caffe2/core/nomnigraph/op_gen.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import absolute_import
 from __future__ import division
--- a/caffe2/python/allcompare_test.py
+++ b/caffe2/python/allcompare_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import absolute_import
 from __future__ import division
--- a/caffe2/python/benchmark_generator.py
+++ b/caffe2/python/benchmark_generator.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import absolute_import
 from __future__ import division
--- a/caffe2/python/caffe_translator.py
+++ b/caffe2/python/caffe_translator.py
@@ -1,6 +1,6 @@
 ## @package caffe_translator
 # Module caffe2.python.caffe_translator
-#!/usr/bin/env python2
+#!/usr/bin/python2
 
 import argparse
 import copy
--- a/caffe2/python/operator_test/alias_with_name_test.py
+++ b/caffe2/python/operator_test/alias_with_name_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 import caffe2.python.hypothesis_test_util as hu
 import hypothesis.strategies as st
--- a/caffe2/python/test/inference_lstm_op_test.py
+++ b/caffe2/python/test/inference_lstm_op_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 import inspect
 
 import hypothesis.strategies as st
--- a/docs/caffe2/process.py
+++ b/docs/caffe2/process.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/python2
 ## @package process
 # Module doxygen.process
 # Script to insert preamble for doxygen and regen API docs
--- a/scripts/build_tizen.sh
+++ b/scripts/build_tizen.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 ##############################################################################
 #  Example command to build the Tizen target (RPi3).
 ##############################################################################
--- a/scripts/release/promote/common_utils.sh
+++ b/scripts/release/promote/common_utils.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 
 exit_if_not_on_git_tag() {
     # Have an override for debugging purposes
--- a/scripts/release/promote/conda_to_conda.sh
+++ b/scripts/release/promote/conda_to_conda.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 
 set -eou pipefail
 
--- a/scripts/release/promote/libtorch_to_s3.sh
+++ b/scripts/release/promote/libtorch_to_s3.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 
 set -eou pipefail
 
--- a/scripts/release/promote/wheel_to_pypi.sh
+++ b/scripts/release/promote/wheel_to_pypi.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 
 set -eou pipefail
 
--- a/scripts/release/promote/wheel_to_s3.sh
+++ b/scripts/release/promote/wheel_to_s3.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/bash
 
 set -eou pipefail
 
--- a/scripts/run_mobilelab.py
+++ b/scripts/run_mobilelab.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 """
 Script to launch all caffe2 MobileLab tests
--- a/test/distributed/rpc/jit/test_dist_autograd_spawn.py
+++ b/test/distributed/rpc/jit/test_dist_autograd_spawn.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 import unittest
 
 from torch.testing._internal.common_distributed import MultiProcessTestCase
--- a/test/distributed/rpc/jit/test_rpc_spawn.py
+++ b/test/distributed/rpc/jit/test_rpc_spawn.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 import unittest
 
 from torch.testing._internal.common_distributed import MultiProcessTestCase
--- a/test/distributed/rpc/test_dist_autograd_spawn.py
+++ b/test/distributed/rpc/test_dist_autograd_spawn.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 import unittest
 
 from torch.testing._internal.common_distributed import MultiProcessTestCase
--- a/test/distributed/rpc/test_dist_optimizer_spawn.py
+++ b/test/distributed/rpc/test_dist_optimizer_spawn.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 from torch.testing._internal.distributed.rpc.dist_optimizer_test import DistOptimizerTest
--- a/test/distributed/rpc/test_rpc_spawn.py
+++ b/test/distributed/rpc/test_rpc_spawn.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 import unittest
 
 from torch.testing._internal.common_distributed import MultiProcessTestCase
--- a/test/run_test.py
+++ b/test/run_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import print_function
 
--- a/test/scripts/run_cuda_memcheck.py
+++ b/test/scripts/run_cuda_memcheck.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 """This script runs cuda-memcheck on the specified unit test. Each test case
 is run in its isolated process with a timeout so that:
--- a/tools/amd_build/build_amd.py
+++ b/tools/amd_build/build_amd.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 from __future__ import absolute_import, division, print_function
 import os
--- a/tools/clang_format.py
+++ b/tools/clang_format.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 A script that runs clang-format on changes detected via git. It will
 report if running clang-format generated any changes.
--- a/tools/clang_format_new.py
+++ b/tools/clang_format_new.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 A script that runs clang-format on all C/C++ files in CLANG_FORMAT_WHITELIST. There is
 also a diff mode which simply checks if clang-format would make any changes, which is useful for 
--- a/tools/clang_tidy.py
+++ b/tools/clang_tidy.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """
 A driver script to run clang-tidy on changes detected via git.
 
--- a/tools/flake8_hook.py
+++ b/tools/flake8_hook.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 import sys
 
--- a/torch/utils/hipify/hipify_python.py
+++ b/torch/utils/hipify/hipify_python.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 """ The Python Hipify script.
 ##
 # Copyright (c) 2015-2016 Advanced Micro Devices, Inc. All rights reserved.
--- a/caffe2/contrib/aten/gen_op.py
+++ b/caffe2/contrib/aten/gen_op.py
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/usr/bin/python3
 
 # Copyright (c) 2016-present, Facebook, Inc.
 #
